nursery/packed-with-themida.yml

```diff
@@ -7,13 +7,20 @@ rule:
     att&ck:
       - Defense Evasion::Obfuscated Files or Information::Software Packing [T1027.002]
     mbc:
-      - Anti-Static Analysis::Software Packing [F0001]
+      - Anti-Static Analysis::Software Packing::Themida [F0001.011]
     references:
       - https://www.hexacorn.com/blog/2016/12/15/pe-section-names-re-visited/
     examples:
+      - 8a132663bee5c2f0f5cbfebee1b55ac72934632bf32bc32d6e2dae797c9e6e35
+      - 2826b762b9c268601a44974ef469a671b441e798a6c3cbb40070450c6c030ba2
   features:
     - or:
       - section: Themida
       - section: .Themida
       - section: .themida
       - section: WinLicen
+      - section: .winlice
+      - count(section(        )): 2 or more
```

Comment:
can you add a description or comment here describing the whitespace? is it 8 spaces?

Reply:
Sure, I added relevant descriptions

Reply:
perfect, thanks!